deploy/helm/azure-industrial-iot/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: azure-industrial-iot
-version: 0.3.0
+version: 0.3.1
```

Comment:
We have already released `0.3.1` version of the chart, so let's make this `0.3.2`.